packages/picasso.js/src/core/chart-components/point/point.js

```diff
@@ -1,15 +1,17 @@
 import extend from 'extend';
 import { notNumber } from '../../utils/is-number';
 
-const DEFAULT_ERROR_SETTINGS = {
-  errorShape: {
-    shape: 'saltire',
-    width: 2,
-    size: 0.5,
-    fill: '#333',
-    stroke: '#333',
-    strokeWidth: 0,
-  },
+/**
+ * @typedef {object}
+ * @alias ComponentPoint.settings.errorShape
+ */
+const ERROR_SHAPE = {
+  /** Type of shape
+   * @type {DatumString=} */
```

Comment:
DatumString and DatumNumber means that they are resolved either as a string or number OR as a function. Which I fairly sure they are not. Just use a string and/or number.

Reply:
By function I mean a `DatumConfig` or a  `datumAccessor`